docs/csharp/language-reference/compiler-messages/cs8410.md

```diff
@@ -0,0 +1,43 @@
+---
+title: Compiler Error CS8410
+ms.date: 07/11/2020
+f1_keywords:
+  - "CS8410"
+helpviewer_keywords:
+  - "CS8410"
+author: Youssef1313
+---
+# Compiler Error CS8410
+
+'type': type used in an asynchronous using statement must be implicitly convertible to 'System.IAsyncDisposable' or implement a suitable 'DisposeAsync' method.
+
+The expression inside an `await using` statement must have a `DisposeAsync` method.
+
+## To correct this error
+
+Remove the `await using` keywords, or implement a suitable `DisposeAsync` method.
+
+## Example
+
+```csharp
+using System.Threading.Tasks;
+
+static class Program
```

Comment:
```suggestion
class Program
```

Reply:
I think it's okay to keep the static keyword? that class contains only a static main method.

Reply:
But it is not necessary. In fact to be consistent with the templates of a new C# console application, it should be omitted.